src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -9,7 +9,25 @@
   <PropertyGroup>
     <Nullable>enable</Nullable>
     <IsOSXLike Condition="'$(TargetsOSX)' == 'true' or '$(TargetsiOS)' == 'true' or '$(TargetstvOS)' == 'true'">true</IsOSXLike>
+    <SupportsArmIntrinsics Condition="'$(Platform)' == 'arm64'">true</SupportsArmIntrinsics>
+    <SupportsX86Intrinsics Condition="'$(Platform)' == 'x64' or ('$(Platform)' == 'x86' and '$(TargetsUnix)' != 'true')">true</SupportsX86Intrinsics>
```

Comment:
the `'$(Platform)' == 'x86' and '$(TargetsUnix)' != 'true'` part only applies to CoreCLR since you don't build an x86 variant on Unix right?

Reply:
I assume it applies to both since this is the existing condition that decides whether to include the `*.PlatformNotSupported.cs` files:

https://github.com/dotnet/runtime/blob/db70edd145a6c9d4825532edcdd27f01999baf97/src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems#L1789-L1790

https://github.com/dotnet/runtime/blob/db70edd145a6c9d4825532edcdd27f01999baf97/src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems#L1807-L1808

Reply:
I think that was just not updated when Mono moved into the repo, but yeah since this is preexisting we don't need to fix it right now.